package.json

```diff
@@ -31,8 +31,11 @@
     "prepare": "husky",
     "prepublishOnly": "npm run build:production",
     "test": "npm run test:unit",
-    "test:unit": "tap --ts tests/test_*.ts",
-    "test:unit-snapshot": "tap --ts --snapshot tests/test_*.ts"
+    "test:cov": "jest --selectProjects unit functional --coverage",
+    "test:unit": "tap --ts tests/test_*.ts && jest --selectProjects unit",
```

Comment:
Were you planning on remove/replacing tap in an another iteration?

Reply:
I'm thinking that these are at least some tests. Replicating them with jest would def take some time. So I decided it's better to leave them till we have more coverage with jest.

Reply:
Ok we can keep it for now and refactor them to jest.